src/modules/admin/components/Profile/ColonyAvatarUploader.tsx

```diff
@@ -49,8 +37,14 @@ const ColonyAvatarUploader = ({ colony: { colonyAddress }, colony }: Props) => {
     ),
     [colonyAddress],
   );
-  const upload = useAsyncFunction({ ...uploadActions, transform }) as any;
-  const remove = useAsyncFunction({ ...removeActions, transform }) as any;
+  const upload = useAsyncFunction({
```

Comment:
We still need the success and error actions here, to have the async function complete